backend/egl.c

```diff
@@ -99,7 +112,7 @@ static bool egl_get_config(EGLDisplay disp, EGLConfig *out, EGLenum platform) {
 			continue;
 		}
 
-		if (gbm_format == GBM_FORMAT_ARGB8888) {
+		if (gbm_format == GBM_FORMAT_XRGB8888) {
```

Comment:
Thanks for the notice, wrote this a while ago and did not realize that the egl code had changed in the meantime.